src/Symfony/Component/Cache/Traits/RedisTrait.php

```diff
@@ -211,11 +212,28 @@ public static function createConnection(string $dsn, array $options = [])
                     if (!isset($params['redis_sentinel'])) {
                         break;
                     }
-                    $extra = [];
-                    if (\defined('Redis::OPT_NULL_MULTIBULK_AS_NULL') && isset($params['auth'])) {
-                        $extra = [$params['auth']];
+
+                    if (version_compare(phpversion('redis'), '6.0.0') >= 0) {
+                        $options = [
+                            'host' => $host,
+                            'port' => $port,
+                            'connectTimeout' => $params['timeout'],
+                            'persistent' => $params['persistent_id'],
```

Comment:
There is no `connect` or `pconnect` call using `RedisSentinel`. So the decision of being persistent or not is being take by this value being `null` or set to a `string`. Not sure why it was converted to `string` before, but this works perfectly fine on my environment at least.